roadmap/implementors-guide/src/types/runtime.md

```diff
@@ -34,5 +34,11 @@ struct HostConfiguration {
 	pub thread_availability_period: BlockNumber,
 	/// The amount of blocks ahead to schedule parathreads.
 	pub scheduling_lookahead: u32,
+	/// Total number of individual messages allowed in the parachain -> relay-chain message queue.
+	pub max_upwards_queue_count: u32,
+	/// Total size of messages allowed in the parachain -> relay-chain message queue before which
+	/// no further messages may be added to it. If it exceeds this then the queue may contain only
+	/// a single message.
+	pub watermark_queue_size: u32,
```

Comment:
Should we choose a name that explicitly refers that it's upward messages queue? I am worried about potential clash with the term watermark that comes from XCMP.